packages/test/jest.test.js

```diff
@@ -0,0 +1,97 @@
+const hrp = require("http-request-plus").default;
+const jrp = require("json-rpc-protocol");
+const assert = require("assert");
+
+async function call(method, params) {
+  return jrp.parse(
+    await hrp
+      .post("http://localhost:4000/api/", {
+        body: jrp.format.request(0, method, params),
+      })
+      .readAll("utf-8")
+  );
+}
+
+test("create entry", async () => {
+  await call("createEntry", { name: "name1", content: "content1" });
+  await call("createEntry", { content: "content2" });
+  await call("createEntry", { name: "name3" });
```

Comment:
While we're at it, test the return values of all calls.

Reply:
should I test the "true" values ​​returned by update and delete, or not?